src/Nethermind/Nethermind.Runner/configs/chiado.cfg

```diff
@@ -0,0 +1,46 @@
+{
+  "Init": {
+    "MemoryHint": 768000000
+    "ChainSpecPath": "chainspec/chiado.json",
+  },
+  "Sync": {
+    "FastSync": true,
+    "FastBlocks": true,
+    "UseGethLimitsInFastBlocks": false,
+    "FastSyncCatchUpHeightDelta": 10000000000
+  },
+  "JsonRpc": {
+    "Enabled": true,
+    "Timeout": 20000,
+    "Host": "127.0.0.1",
+    "Port": 8545,
+    "AdditionalRpcUrls": [
+      "http://localhost:8551|http;ws|net;eth;subscribe;engine;web3;client"
+    ]
+  },
+  "AuRaMerge": {
+    "Enabled": true
+  },
+  "Aura": {
+    "TxPriorityContractAddress": "0x4100000000000000000000000000000000000000",
+    "ForceSealing": true,
+    "AllowAuRaPrivateChains": true
```

Comment:
is it a private chain now? I think not.

Reply:
Please adapt the flags to what makes sense, ansible config can be sloppy with those, lots of bagage from previous runs so don't copy what's there blindly